### ISSUE TITLE
Sometimes it is necessary to disable the reporting of window title changes by NVDA

### ISSUE THREAD

**Issue by shadyar** [nvaccess/nvda#17520]
### Steps to reproduce:
Sometimes it is necessary to disable the reporting of window title changes by NVDA.
For example When you are studying a web page that its title is constantly changes, NVDA reports the title change every 5 secs. It is really annoying.
like this web page: https://www.binance.com/en/trade/BTC_USDT?type=spot

### Actual behavior:

NVDA reports the title change every 5 secs. It is really annoying.
### Expected behavior:

A feature should be added to settings to enable / disable window title changes report.
#### NVDA version:
2024.4.1 (2024.4.1.34550)

#### Windows version:
Windows 11 64-bit updated version.
#### Name and version of other software in use when reproducing the issue:
Firefox updated version.
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by seanbudd** [comment]
To fix this, this should be disabled when toggling "report dynamic content changes"

**Comment by shadyar** [comment]
@SaschaCowley 
No, the title changes of Mozilla Firefox is read even while the "report dynamic content changes" is disabled. Please test it by yourself, open the link I written by Firefox.
I disabled it by pressing NVDA + 5 I had tested this solution before I created this issue.

**Comment by SaschaCowley** [comment]
@shadyar, we are aware that this is broken. What my colleague was saying is that in order to fix this, we need to add title bar changes to those ignored when dynamic content change reporting is disabled. Apologies for the confusion.

**Comment by shadyar** [comment]
this issue can even unwittingly occurred in other applications such as players etc that I did not necessarily come across.this issue can even unwittingly occurred in other applications such as players etc that I did not necessarily come across.

**Comment by shadyar** [comment]
@SaschaCowley 
You're welcome. Thanks so much for your attention.

**Comment by XLTechie** [comment]
I was just very annoyed by this one a few minutes before coming across this issue for the first time.

I happened to try to use context help in NVDA, to confirm that an Advanced setting did what I thought it did. It opened the user guide in Firefox, which must have crashed recently. Consequently, it began loading all my dozens of open tabs.

With each tab loaded, or window re-opened, it apparently re-drew the title of the NVDA user guide page. That made reading practically impossible, until I switched to on-demand speech mode, and used review keys to read the desired section.

Firefox is the biggest offender I've personally noticed in this regard.
I have no idea what might be going on visually, that causes the title to re-draw for each background task launch, but maybe it's shrinking to show more tabs?

Very in favor of fixing this one!

**Comment by Adriani90** [comment]
Cc: @jcsteh  at least for Firefox.

**Comment by jcsteh** [comment]
Note that NVDA only reports changes to the title of a document or window if that document or window itself (not a descendant) has focus. This isn't to suggest that the bug is any less valid, just that the bug is more specific than just any change to a window title.

> With each [Firefox] tab loaded, or window re-opened, it apparently re-drew the title of the NVDA user guide page.

@XLTechie, can you still reproduce this? If so, could you get me an NVDA debug log? I can't think of any reason Firefox should be firing name change events on the User Guide document in this case. We only do that if the document title changes, which it isn't. I'm wondering whether you were seeing a different issue where focus was shifting between tabs or something like that. That's equally annoying, but a different issue which needs a different solution.

**Comment by shadyar** [comment]
Ok, I sent a log, but you yourself can reproduce easily this issue by opening the link I mentioned in this issue through Firefox.
I write the link again here:
https://www.binance.com/en/trade/BTC_USDT?type=spot
[nvda.log](https://github.com/user-attachments/files/18297121/nvda.log)
